runtime/etc/Makefile.mli-static

```diff
@@ -23,15 +23,31 @@ include $(CHPL_MAKE_HOME)/runtime/etc/Makefile.include
 
 CHPL_MLI_INCS = -I$(CHPL_ZMQ_LOC)/include
 CHPL_LN_LIB_DIR = $(CHPL_MAKE_RUNTIME_LIB)/$(CHPL_MAKE_LAUNCHER_SUBDIR)
+UNAME = $(shell uname)
+CHPL_LIBTOOL = libtool
+CHPL_LIBTOOL_CMD =
 
-.PHONY: buildUserObj buildServer buildClient runTagsCommand
+.PHONY: setLibtoolCmd buildUserObj buildServer buildClient runTagsCommand
 
-all: buildUserObj buildServer buildClient runTagsCommand FORCE
+all: setLibtoolCmd buildUserObj buildServer buildClient runTagsCommand FORCE
+
+#
+# On some platforms (Darwin in particular), libtool cannot be assumed to mean
+# GNU libtool and is actually a different executable that goes by the same
+# name. Darwin's libtool has similar semantics to GNU libtool, but has to be
+# invoked differently.
+#
+setLibtool:
+ifeq ($(UNAME),Darwin)
```

Comment:
I think it would be better to use `CHPL_MAKE_TARGET_PLATFORM` rather than performing an independent check.  In that case, it will be `darwin` rather than `Darwin`

Reply:
Yes, Elliot said much the same, which probably means all of the Makefiles should be changed.

Reply:
As in Makefile.mli-shared?  What other Makefiles were you thinking?